client/deposits/list/index.tsx

```diff
@@ -250,7 +257,7 @@ export const DepositsList = (): JSX.Element => {
 				try {
 					const {
 						exported_deposits: exportedDeposits,
-					} = await apiFetch( {
+					} = await apiFetch< { exported_deposits: number } >( {
```

Comment:
See https://github.com/Automattic/woocommerce-payments-server/blob/2fa0b7aedc83386b4fcfe399adebd8396ea3461e/server/wp-content/rest-api-plugins/endpoints/wcpay/service/class-deposit-service.php#L739 for source of response.